eemeter/modeling/models/hourly_load_profile.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import datetime
+import eemeter.modeling.exceptions as model_exceptions
+from eemeter.modeling.models.caltrack_daily import CaltrackDailyModel
+
+
+class HourlyLoadProfileModel(object):
```

Comment:
We should get some basic tests on this before merging. Take a look at the other models for examples. Doesn't need to be too complicated.